cloud/blockstore/libs/rdma/impl/client.cpp

```diff
@@ -952,7 +951,21 @@ void TClientEndpoint::SendRequest(TRequestPtr req, TSendWr* send)
     requestMsg->Out = req->OutBuffer;
 
     RDMA_TRACE("SEND " << TWorkRequestId(send->wr.wr_id));
-    Verbs->PostSend(Connection->qp, &send->wr);
+
+    try {
+        Verbs->PostSend(Connection->qp, &send->wr);
+    } catch (const TServiceError& e) {
+        RDMA_ERROR(
+            "SEND " << TWorkRequestId(send->wr.wr_id) << ": " << e.what());
+        SendQueue.Push(send);
+        ReportRdmaError();
+        Disconnect();
```

Comment:
What about the server.cpp it also has post_send and post_recv which can leak resources in the same way. Let's fix server.cpp as well 

for PostSend we can call ReadRequestDataCompleted/WriteResponseDataCompleted/SendResponseCompleted
for PostRecv the RecvRequestCompleted tries to PostRecv again so it's more tricky probably need to flush session

Reply:
This PR is specifically about the client. Server fix will require more work and will probably come quite a bit later